server/package.json

```diff
@@ -49,7 +49,7 @@
     "@swc/core": "^1.2.163",
     "@types/express": "^4.17.13",
     "@types/jest": "^27.4.1",
-    "@types/node": "^16.11.7",
+    "@types/node": "^22.7.5",
```

Comment:
Pour fix `Error: node_modules/@types/node/globals.d.ts(72,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; abort(reason?: any): AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.`

Reply:
c'est étonnant qu'on ne l'avait pas déjà